packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -1804,6 +1813,213 @@ describe("IModelTransformerHub", () => {
     }
   });
 
+  it.only("should propagate custom inserts and custom deletes", async () => {
```

Comment:
More tests will be needed e.g. to add / delete model

Reply:
I feel like it is easier to add tests on my side, as I have "filtered" target. Here it is more  complicated, eg: to test insert you need to delete that element from target first. Is there a way to simplify this?

Reply:
I suppose instead of deleting it from target, I could exclude the things I wnat to later add as custom changes using excludeElement? Would have to expose that in the timelinetestutil.ts I think though. I could see that making the test a bit 'simpler' but idk by how much.

When you say you have the 'filtered' target. Do you mean you are pushing to the repo some small test iModel? Maybe even two? the source and the filtered target?

I'm not opposed to having something like that as I know other repos do it like itwinjs-core. If we were to do it though im unsure about where to draw the line. It'd probably be overkill to do it for every test of course but maybe here and there seems reasonable